linera-views/src/batch.rs

```diff
@@ -523,9 +504,7 @@ impl BatchValueWriter<SimpleUnorderedBatch> for SimpleUnorderedBatchIter {
                 + get_uleb128_size(batch.deletions.len())
                 + get_uleb128_size(batch.insertions.len() + 1))
         } else {
-            Ok(batch_size
-                + get_uleb128_size(batch.deletions.len())
-                + get_uleb128_size(batch.insertions.len()))
+            unreachable!();
```

Comment:
I would either return an error or better change the result of the method to be `Result<Option<usize>, bcs::Error>`.

Reply:
Ok, this has done as you requested.